WordPress/Jetpack/Classes/Utility/MigrationHelper.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+struct MigrationHelper {
```

Comment:
Please feel free to brainstorm a better name, as Helpers are generally code smells. 😄

Reply:
@twstokes If you decide to keep the `MigrationHelper.swift` file, then I think it would be great to move it closer to other migration files.

<img width="352" alt="CleanShot 2022-11-28 at 12 12 51@2x" src="https://user-images.githubusercontent.com/9609223/204264000-8954aeeb-14b2-44fe-90f3-fe9cd168dd9e.png">

Reply:
Thanks for the suggestions @salimbraksa!

Right now we're looking for the presence of the WordPress app, and potentially in a [future PR](https://github.com/wordpress-mobile/WordPress-iOS/issues/19656) we're going to check if a scheme exists that supports migrations.

Using this model we'd add a new `AppScheme` like `jetpacknotificationmigration` to support that detection. Does that sound reasonable to you?

Reply:
>Using this model we'd add a new AppScheme like `jetpacknotificationmigration`  to support that detection.

I think you meant `wordpressnotificationmigration`, right? If that's the case, then it makes sense to have a specific scheme for WordPress builds that support migration. 

Also, the purpose of `AppScheme` is to access all app schemes ( the ones in `Info.plist` ) in a type-safe way. For now, it's okey to have only `wordpress` / `wordpressnotificationmigration` or both. But eventually, it would be great to have all app schemes in that enum.

Reply:
> I think you meant wordpressnotificationmigration, right? If that's the case, then it makes sense to have a specific scheme for WordPress builds that support migration.

Yep, apologies. [This one](https://github.com/wordpress-mobile/WordPress-iOS/pull/19616/files#diff-4853625f4a22ad571bda67f42388cfda828d488cdccb890657caad3285eefa41R480).

> Also, the purpose of AppScheme is to access all app schemes ( the ones in Info.plist ) in a type-safe way. For now, it's okey to have only wordpress / wordpressnotificationmigration or both. But eventually, it would be great to have all app schemes in that enum.

That makes sense. 👍 

I've pushed the updates and updated the testing steps if you'd like to give it another look. Thanks!

Reply:
LGTM! 🚀

Reply:
> since the type AppScheme is now feature-agnostic, perhaps we can move the file to feature-agnostic location?

Yeah that sounds good. Initially I had the intention of keeping this grouped with the migration code since it's destined for removal, but perhaps we'll utilize the `canOpen` functionality in the future. If we can't identify a good reason to keep it once the migration is over we can wipe it then.

Reply:
Thank you and I agree. We might add some Todo in there (or create a map elsewhere) or anything that needs removal after the migration phase is over.